Model/Carrier/LogitrailCarrier.php

```diff
@@ -71,11 +71,16 @@ public function getForm($lang = 'fi')
     {
         $this->session->setLogitrailShippingCost(0);
 
-        $items = $this->quote->getAllVisibleItems();
+        $items = $this->quote->getAllItems();
         $api = $this->logitrail->getApi();
         //$api->setOrderId($this->quote->getId());
         $api->setOrderId($this->quote->getId());
+
+        /** @var \Magento\Quote\Model\Quote\Item $item */
         foreach ($items as $item) {
+            if (count($item->getChildren()) > 0) {
```

Comment:
I suspect this is here to handle bundled products, but for clarity again, it could be told also for those, who hasn't been included in conversations about this.